ee/dtr/admin/configure/external-storage/storage-backend-migration.md

```diff
@@ -0,0 +1,78 @@
+
+### Best practice for data migration
+
+Docker recommends the following steps for your storage backend and metadata migration:
+
+1. Disable garbage collection by selecting "Never" under **System > Garbage Collection**, so blobs referenced in the backup that you create continue to exist. See [Garbage collection](/ee/dtr/admin/configure/garbage-collection/) for more details.
+
+    ![](/ee/dtr/images/garbage-collection-0.png){: .img-fluid .with-border}
+
+2. [Back up your existing metadata](/ee/dtr/admin/disaster-recovery/create-a-backup/#back-up-dtr-metadata). See [docker/dtr backup](/reference/dtr/2.6/cli/backup/) for CLI command description and options. 
+
+3. Migrate the contents of your current storage backend to the new one you are switching to. For example, upload your NFS contents to an S3 bucket if you're switching from NFS to S3.
+
+4. [Restore DTR from your backup](/ee/dtr/admin/disaster-recovery/restore-from-backup/) and specify your new storage backend. See [docker/dtr destroy](/reference/dtr/2.6/cli/destroy/) and [docker/dtr restore](/reference/dtr/2.6/cli/backup/) for CLI command descriptions and options.
+
+5. Run garbage collection in "blob" mode. You can either SSH into a DTR node, or [use a UCP client bundle](/ee/ucp/user-access/cli/) to run the following command:
+
+    ```bash
+    docker exec -it dtr-jobrunner-<replica_id> sh
+    ```
+    > See [Find your replica ID](/ee/dtr/admin/disaster-recovery/create-a-backup/#find-your-replica-id) for tips on determining your replica ID.
+
+    Within the running container, type `/bin/job_executor` to start the `job_executor` binary.
+    
+    ```bash
+    /bin/job_executor
+    
+    / # onlinegc blobs
+    ```
+    > Note that the first line results in a display of the binary name, its usage, and available commands including `onlinegc`. To learn more about a command, run `<command_name>` --help.
+
+    Running garbage collection in blob mode destroys any new blobs which are not referenced in your previously created backup.
+    
+### Alternative options for data migration
+
+- If you have a long maintenance window, you can skip some steps from above and do the following:
+
+    1. Put DTR in "read-only" mode. To do so, send the following API request:
+   
+        ```bash
+        curl -u <username>:$TOKEN -X POST "https://<dtr-url>/api/v0/meta/settings" -H "accept: application/json" -H "content-type: application/json" -d "{ \"readOnlyRegistry\": true }"
+        ``` 
+        On success, you should get a `202 Accepted` response.
+
+    2. Migrate the contents of your current storage backend to the new one you are switching to. For example, upload your NFS contents to an S3 bucket if you're switching from NFS to S3.
+
+   3. [Reconfigure DTR](/reference/dtr/2.6/cli/reconfigure) while specifying the `--storage-migrated` flag to preserve your existing tags. 
+
+- If you are not worried about inconsistencies in data, skip steps 1 and 2 and perform a reconfigure.
```

Comment:
I don't know if I am understanding correctly. 

Is this basically saying, if you don't care about your images then you can run a reconfigure?

Data inconsistencies makes it sound like data will be retained, but it could be wrong. 

Is the below comment indicating that there are data inconsistencies? 

> Starting in DTR 2.6, switching storage backends initializes a new metadata store and erases your existing tags

Or is this comment indicating that your images will get deleted?